.github/workflows/python.yml

```
@@ -36,33 +39,18 @@ jobs:
           git config user.email "<>"
           git --version
 
-      - name: Package Python 3.8 and earlier
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.8' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io ezralanglois/interop sh /io/tools/package.sh /io /io/dist travis OFF
+      - name: Package Python 3.9-3.x Linux x86_64
```

Update also this since you're removing the step for 3.8 above
```suggestion
      - name: Package Python 3.8-3.x Linux x86_64
```

---

Good point